internal/services/datafactory/data_factory_integration_runtime_azure_ssis_resource.go

```diff
@@ -638,9 +645,16 @@ func expandDataFactoryIntegrationRuntimeAzureSsisProperties(d *pluginsdk.Resourc
 	if catalogInfos, ok := d.GetOk("catalog_info"); ok && len(catalogInfos.([]interface{})) > 0 {
 		catalogInfo := catalogInfos.([]interface{})[0].(map[string]interface{})
 
+		var pricingTier datafactory.IntegrationRuntimeSsisCatalogPricingTier
```

Comment:
Can we add some comments about the fact that both `pricing_tier` and `elastic_pool_name` ends up to the same API prop: `CatalogPricingTier`